docs/appendix/glossary/resource-config.md

```diff
@@ -2,8 +2,10 @@
 title: Resource Config
 id: resource-config
 full_link:
-short_description: The configuration element of either a component or a service. Typically expressed in JSON.
+short_description: The configuration element of either a component or a service.
 aka:
 ---
 
-The configuration element of either a {{< glossary_tooltip term_id="component" text="component" >}} or a service. Typically expressed in JSON.
+The configuration element of either a {{< glossary_tooltip term_id="component" text="component" >}} or a {{< glossary_tooltip term_id="service" text="service" >}}.
```

Comment:
This glossary item is strange. I would think this term could also apply to the entire config, like "the config of all the robot's resources". I almost think we should just delete this term. If not, it could still apply to a remote config, a module config, etc., so maybe broaden it.
```suggestion
The configuration element of a {{< glossary_tooltip term_id="resource" text="resource" >}} such as a {{< glossary_tooltip term_id="component" text="component" >}} or a {{< glossary_tooltip term_id="service" text="service" >}}.
```